edx-platform/pearson-vue-theme/lms/student-account/form_field.underscore

```diff
@@ -133,6 +133,6 @@
     <% } %>
 
     <% if( form === 'login' && name === 'password' ) { %>
-        <button type="button" class="forgot-password field-link"><%- gettext("Need help logging in?") %></button>
+        <button onclick="changeTitle()" type="button" class="forgot-password field-link"><%- gettext("Need help logging in?") %></button>
```

Comment:
Could you add an explanatory comment on this and on the function itself? @luismorenolopera

Reply:
done @Squirrel18